packages/components/text-field/src/text-field.stories.ts

```diff
@@ -15,7 +15,6 @@ const sizes: TextFieldSize[] = ['md', 'lg'];
 export const API: StoryObj = {
```

Comment:
Can you please also improve min max length story? Validation message is not shown after validation clicked
<img width="616" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/35f276c4-8108-4cc7-9cab-0b6e617d4923">

Reply:
Also Pattern story:

<img width="675" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/836edd0d-8538-47c2-847f-144123964de9">

Reply:
And also that story with Custom validation?
<img width="643" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/cafa2ff9-50b7-422f-91a6-41e5313c0796">

Reply:
I've updated the stories